fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/FedoraTransaction.java

```diff
@@ -34,4 +34,9 @@
      */
     void rollback();
 
+    /**
+     * Get the transaction id
+     */
+    String getId();
```

Comment:
Added the getID to the FedoraTransaction interface for the services to be able to get the id and pass it to the persistence layer. Is there any problem with getId being in the interface?

Reply:
I don't see any issue there.